k-distribution/include/builtin/domains.k

```diff
@@ -863,6 +863,14 @@ endmodule
 
 module BYTES-SYMBOLIC [symbolic, kast]
   imports BYTES-IN-K
+  imports BYTES-SYMBOLIC-CEIL
```

Comment:
this entire module is not going to get imported on the haskell backend, thus, neither will this import.

Reply:
Can you explain how module attributes work? Will `[symbolic, kast]` get imported by union or intersection of backends marked with [symbolic] and [kast]? If it's union, I guess hashell backend will import `BYTES-SYMBOLIC`.

Reply:
A module tagged `symbolic, kast` will be imported only by the java backend.

Reply:
Ok, so what's your solution, where to put those rules? I have no idea based on your explanation. Feel free to close the PR/fix/suggest an actionable fix.

Reply:
It's the intersection of the `kore` and `symbolic` which makes it the Haskell backend.

So you may need to make a new module.